diff --git a/node_modules/toggl-api/lib/client.js b/node_modules/toggl-api/lib/client.js
index 9c2511f..c7dab51 100644
--- a/node_modules/toggl-api/lib/client.js
+++ b/node_modules/toggl-api/lib/client.js
@@ -13,8 +13,8 @@ var DEFAULT_INSTANCE = null;
 var DEFAULTS = {
   reauth: false,
   sessionCookie: 'toggl_api_session',
-  apiUrl: 'https://www.toggl.com',
-  reportsUrl: 'https://toggl.com/reports'
+  apiUrl: 'https://api.track.toggl.com',
+  reportsUrl: 'https://api.track.toggl.com/reports'
 };
 
 
